metis-dereference/metis-dereference-import/src/main/java/eu/europeana/metis/dereference/vocimport/utils/DereferenceValidationUtils.java

```diff
@@ -0,0 +1,64 @@
+package eu.europeana.metis.dereference.vocimport.utils;
+
+import edu.emory.mathcs.backport.java.util.Arrays;
```

Comment:
Wrong import here

Reply:
Thanks for pointing it out! I didn't realize this, should pay more attention to this. I will change this